### ISSUE TITLE
Scrollbar color contrast is less than 3:1 in Light 2026 and Dark 2026 themes:A11y_VisualStudioCode_Editor_

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#318703]
Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS;#Visual Studio Code Client; #E2E_VisualStudioCodeClient_Win32_Dec2025;#DesktopApp;#FTP;#Win32;#A11ySev2; #A11yMAS; #WCAG1.4.11; #Non-TextContrast; #SCIM:Bug5119838;

## Issue Subcategory 
Slider or range control 

## User Experience 
Users with cataracts may be unable to locate a volume or zoom slider when the track and thumb are barely visible against the background, making it difficult to identify the current position or drag the control to a new value.

## Repro Steps 
Launch Visual Studio Code > Apply Light 2026/Dark 2026 themes >Create a New Jupyter notebook file > Open Created file > Observe issue: Scrollbar color contrast is less than 3:1 in Light 2026 and Dark 2026 themes

## Actual Result 
The contrast ratio between the scrollbar thumb and its background is less than 3:1, making the scrollbar difficult to perceive.

## Expected Result 
The scrollbar thumb should have a contrast ratio of at least 3:1 against the adjacent background, ensuring it is clearly visible.

## How to Fix 
Darken the track and thumb colors; ensure both parts meet 3:1 contrast. 

## Environment details 
Application Name: VS  Code
Version: 1.121.0
Edition Windows 11 Enterprise
Version 25H2
OS build 26200.8390

## Attachment

<img width="1268" height="516" alt="Image" src="https://github.com/user-attachments/assets/bd9d7cfe-1a65-4704-8116-6269c6e30acc" />
<img width="1279" height="497" alt="Image" src="https://github.com/user-attachments/assets/1856ca7a-23c6-4449-aa05-e77781b6e5a6" />

**Comment by mrleemurray** [comment]
@kapilvaishna please verify the fix!

**Comment by kapilvaishna** [comment]
@mrleemurray In Light 2026 theme still color  contrast is les than 3:1 for scrollbar

<img width="1280" height="768" alt="Image" src="https://github.com/user-attachments/assets/91752fac-6ce8-4e87-baef-42a2ff597cd2" />

<img width="1280" height="768" alt="Image" src="https://github.com/user-attachments/assets/ad942f76-15e9-426e-a3c2-0646e63b3b69" />

**Comment by meganrogge** [comment]
@mrleemurray should this be closed?

**Comment by jruales** [comment]
Verified on macOS
